flask_monitoringdashboard/database/function_calls.py

```diff
@@ -27,8 +27,11 @@ def add_function_call(time, endpoint, ip):
     """ Add a measurement to the database. """
     with session_scope() as db_session:
         group_by = None
-        if config.get_group_by:
-            group_by = config.get_group_by()
+        try:
+            if config.get_group_by:
```

Comment:
the last think i'll ask on this PR: 
please reverse the order: if config then try catch.